etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -24,6 +24,7 @@ var (
 	ErrGRPCEmptyKey     = grpc.Errorf(codes.InvalidArgument, "etcdserver: key is not provided")
 	ErrGRPCKeyNotFound  = grpc.Errorf(codes.InvalidArgument, "etcdserver: key not found")
 	ErrGRPCValue        = grpc.Errorf(codes.InvalidArgument, "etcdserver: value is provided")
+	ErrGRPCLease        = grpc.Errorf(codes.InvalidArgument, "etcdserver: lease is provided")
```

Comment:
@xiang90 Surre will change it. Let me know if any other change also required will fix them all at once.
Thanks!